Assets/Scripts/Enemy/StateMachine.cs

```diff
@@ -111,4 +112,34 @@ public void ExitState()
             Debug.Log("Exit Alert state");
         }
     }
+
+    public class DeadState : IStates
+    {
```

Comment:
This is probably not needed as the state machine gets removed when the enemy dies but I may change this in the future